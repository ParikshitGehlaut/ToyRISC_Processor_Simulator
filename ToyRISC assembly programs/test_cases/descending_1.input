	.data
a:
	70
	80
	40
	20
	10
	30
	50
	60
n:
	8
