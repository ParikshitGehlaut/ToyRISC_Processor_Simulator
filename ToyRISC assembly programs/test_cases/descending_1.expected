Main Memory Contents:

0		: 80
1		: 70
2		: 60
3		: 50
4		: 40
5		: 30
6		: 20
7		: 10
